### DISCUSSION TITLE
import '***.css'

### DISCUSSION THREAD

**Discussion by fushengruomengzhang** [FranckFreiburger/vue3-sfc-loader#91] [Q&A] [answered]
* file:test.vue
```vue
<template>test</template>
<script>
import test from '../test.js' 
import '../test.css' // When I use it like this, the console gives an error
...
...
</script>
```
![image](https://user-images.githubusercontent.com/63709269/124342153-854f4500-dbf4-11eb-863f-88a7cd9abf8e.png)

**Comment by FranckFreiburger** [comment, marked as answer]
Hi, have a look at this new example : https://github.com/FranckFreiburger/vue3-sfc-loader/blob/main/docs/examples.md#import-style

**Reply by fushengruomengzhang** [reply]
is ok
